gems.locked

```diff
@@ -1,6 +1,6 @@
 GIT
   remote: https://github.com/ONLYOFFICE/ooxml_parser
-  revision: 2aae1b3e5c3ab9fcb295644cc02a21ad07858d65
+  revision: 17473fef1c98448ee4bf864a3ed51cc39b41103b
```

Comment:
Do not update gems.lock without any reason (I don't think those test need this update)